webapp/components/survey/SurveyDropdown/useSurveyDropdownOptions.js

```diff
@@ -5,11 +5,26 @@ import * as Survey from '@core/survey/survey'
 import * as API from '@webapp/service/api'
 import { useI18n } from '@webapp/store/system'
 
+const getSurveyLabel = (surveyInfo) => {
+  // label will be: survey_name - survey_label
+
+  // survey name in bold (markdown)
+  const parts = [`**${Survey.getName(surveyInfo)}**`]
```

Comment:
what? now the survey label could be written with markdown?

Reply:
what is the reason behind? I saw the ticket, is that a requirement?

Reply:
The survey label cannot be written in markdown.
The requirement is to have the possibility to format somehow the label in the surveys dropdown.
In this way, when the parameter passed to the dropdown markdownInLabels is true, the text in label and description will be considered as markdown syntax.
I think it could be useful even in other cases, like variables dropdown in the expression editor. The alternative is to create a custom component for the option, but this way is easier...